numba/cuda/mathimpl.py

```diff
@@ -249,13 +248,53 @@ def lower_ldexp_impl(context, builder, sig, args):
 impl_ldexp(types.float64, libdevice.ldexp)
 
 
+def impl_tanh(ty, libfunc):
+    def lower_tanh_impl(context, builder, sig, args):
+        def get_compute_capability():
+            cstk = targetconfig.ConfigStack()
+            if cstk:
+                flags = cstk.top()
+                return flags.compute_capability
+            else:
+                return None
```

Comment:
From OOB discussion, I think the conclusion is that the top of the config stack can never be `None` (an `IndexError` would be raised in this case).

```suggestion
            flags = targetconfig.ConfigStack().top()
            return flags.compute_capability
```